install-tbb.bat

```diff
@@ -0,0 +1,9 @@
+@ECHO off
+echo Permanently setting the PATH user environment variable:
+
+for /F "tokens=2* delims= " %%f IN ('reg query HKCU\Environment /v PATH ^| findstr /i path') do set OLD_SYSTEM_PATH=%%g
+setx Path %~dp0stan\lib\stan_math\lib\tbb;%OLD_SYSTEM_PATH%
```

Comment:
If built with cmdstan tbb is build to `cmdstan\lib\tbb` not `cmdstan\stan\lib\stan_math\lib\tbb`. Either fix this path here or make tbb build to the stan_math folder. I would prefer the latter.

Reply:
Other than that I can confirm this works. I like that its much more windows-like than my sed thing.

Reply:
For me the TBB gets build in cmdstan/stan/lib/stan_math/lib/tbb which is the correct location, I think. It has to be under stan_math as it is part of stan_math (and the makefiles for it live there).

Reply:
I will do a clean clone to double check.

Reply:
On windows I get the following:
```
NOTE: Please add C:/Users/Rok/Desktop/cmdstan/lib/tbb to your PATH variable.
You may call

mingw32-make install-tbb
```

On Ubuntu its the same thing. Are you sure its where it gets build on your system? I agree that stan_math is the right place.

Reply:
argh.. found it. Fixing it on the math tbb-lib branch. Sorry.

Reply:
Great. This isnt a problem with this PR so resolving.